src/applications/personalization/view-dependents/manage-dependents/tests/e2e/manage-dependents.cypress.spec.js

```diff
@@ -0,0 +1,91 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import mockDependents from '../fixtures/mock-dependents.json';
+import mockUser from '../fixtures/mockUser.json';
+import { rootUrl } from '~/applications/personalization/view-dependents/manifest.json';
+
+const VIEW_DEPENDENTS_PATH = rootUrl;
+const VIEW_DEPENDENTS_ENDPOINT = '/v0/dependents_applications/show';
+const FORM_686_ENDPOINT = '/v0/dependents_applications';
+
+const setup = () => {
+  disableFTUXModals();
+  cy.login(mockUser);
+  cy.intercept('GET', VIEW_DEPENDENTS_ENDPOINT, mockDependents).as(
+    'mockDependents',
+  );
+  cy.intercept('GET', '/v0/feature_toggles*', {
+    data: {
+      type: 'feature_toggles',
+      features: [
+        {
+          name: 'dependents_management',
+          value: true,
+        },
+      ],
+    },
+  });
+  cy.visit(VIEW_DEPENDENTS_PATH);
+};
+
+const testAxe = () => {
+  cy.injectAxe();
+  cy.axeCheck();
+};
+
+const testInlineValidation = async () => {
+  const buttons = await cy.findAllByRole('button', {
+    name: /Remove this dependent/,
+  });
+  expect(buttons).to.have.length(2);
+  buttons[0].click();
+  await cy
+    .findByRole('button', {
+      name: /Submit VA Form 686c to remove this dependent/,
+    })
+    .click();
+  await cy.findByText(/Please select an option/);
+  cy.findAllByRole('alert').should('have.length', 4);
+  testAxe();
```

Comment:
I've learned the hard way that it makes sense to run an axe check _before_ you start interacting with the page as well as after, since the DOM will likely change.